hmac/decryptor/hashQuery.go

```diff
@@ -198,20 +203,24 @@ func (encryptor *HashQuery) OnBind(ctx context.Context, statement sqlparser.Stat
 			indexes = append(indexes, index)
 		}
 	}
+
+	bindData := queryEncryptor.ParseSearchQueryPlaceholdersSettings(statement, encryptor.schemaStore)
+	if len(bindData) != len(indexes) {
```

Comment:
above we collect only searchable items. but as I understand, `ParseSearchQueryPlaceholdersSettings` will return tokenized items too and they will break the expected amount of items

we should add tests with several security controls, with searchable, tokenized and simple data bound via `EXECUTE` SQL query